src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/SystemLookup.java

```diff
@@ -52,10 +58,15 @@ private SystemLookup() { }
      * on Windows. For this reason, on Windows we do not generate any side-library, and load msvcrt.dll directly instead.
      */
     private static final SymbolLookup syslookup = switch (CABI.current()) {
-        case SysV, LinuxAArch64, MacOsAArch64 -> libLookup(libs -> libs.loadLibrary("syslookup"));
+        case SysV, LinuxAArch64, MacOsAArch64, SysVPPC64le, SysVS390x -> libLookup(libs -> libs.loadLibrary("syslookup"));
+        case AIX -> makeAixLookup();
         case Win64 -> makeWindowsLookup(); // out of line to workaround javac crash
     };
 
+    private static SymbolLookup makeAixLookup() { // Intended for libc.a on AIX
+        throw new InternalError("Default library loading is not yet implemented on AIX"); //$NON-NLS-1$
```

Comment:
is there an issue open for this?

Reply:
Yes, it was created previously at https://github.com/eclipse-openj9/openj9/issues/13679.

Reply:
thanks